### ISSUE TITLE
Research - Auth Profile - MHV Research- Topline Summary

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#39402]
## Background

As a Researcher, I want to create a topline summary to capture the major learnings from research sessions with Stakeholders and Veterans.

## Tasks
- [x] Create topline summary
- [x] Review topline summary with PM
- [x] Deliver topline summary to PO

## Acceptance Criteria
- [x] Create topline summary
- [x] Review topline summary with PM
- [x] Deliver topline summary to PO

## How to configure this issue
- Include practice area in title -- e.g., [Design], [FE], [BE]
- Include project/initiative name - e.g., Auth Profile: Address Change Messaging
- Add label for practice area (frontend, backend, design)
- Add label for project if applicable
- Assign to a Profile team member
- Associate with an Epic if applicable
- If creating for yourself: estimate work if enough info exists to do so

**Comment by Samara-Strauss** [comment]
Totally fine! Take all the time you need.

**Comment by andaleliz** [comment]
Hi @Samara-Strauss - the[ topline summary](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/mhv-profile-research/mhv-user-interviews/topline-summary.md) is ready for your review!  Please let me know if you have any feedback or edits you'd like to see. 

I'll set up some time for us to talk about next steps on Monday.

**Comment by Samara-Strauss** [comment]
@andaleliz this is great!! Really nice work. Thank you for putting this together. Only a few things on my end:

> There were some topics Veterans felt had overlap between healthcare and profile.
> Participants expect to find some health related information in their profile.

The card sort findings are related, and I expected to see them either under a separate "card sort" heading, or consecutively, OR consolidated into one finding (these feel related enough to group as one finding). "MHV users are used to being able to change their password and manage account access via security questions" doesn't seem related to these two findings, yet it was between them in the findings list. 

---

Unrelated to the above, why are the last three findings under "Additional findings?" They don't seem to vary in importance from the other findings, so I wasn't sure why they were separated out.

---

> The participants who receive care at multiple facilities have not experienced challenges related to being registered at multiple VAMCs.
> Some participants shared how the ability to select a facility could be valuable to them:

These feel unintentionally contradictory. Largely, the finding was that people did not need to be able to select or deselect facilities in their profiles, but having the second heading be about how some people want the ability to select this makes it sound like people mostly find it to be valuable feature. Additionally, I think the major finding here is that it would only be valuable if it was affecting other data downstream at the VA. So that should be clarified as well.

Finally, you said "The ability to select or deselect a facility seemed invaluable to some participants...". I *think* you mean that the ability to select or deselect a facility seemed like it was not valuable to some participants, but "invaluable" means "very valuable". So, if you were intending to say that people wouldn't find this valuable, the wording needs to be updated.

Let me know if you want to chat about any of this! I think a few tweaks for clarity are all that's needed. The findings are great and right on to what we heard in the research. Nice job!

**Comment by andaleliz** [comment]
Thanks for your review Samara! 

>The card sort findings are related, and I expected to see them either under a separate "card sort" heading, or consecutively, OR consolidated into one finding (these feel related enough to group as one finding). "MHV users are used to being able to change their password and manage account access via security questions" doesn't seem related to these two findings, yet it was between them in the findings list.

Excellent point; I've reorganized and created a section labeled "Interview findings" and "Card sort findings"

>Unrelated to the above, why are the last three findings under "Additional findings?" They don't seem to vary in importance from the other findings, so I wasn't sure why they were separated out.

I separated those out because they were unrelated to our research goals for this activity, but I felt worth mentioning. I can certainly put them into the interview findings if that's confusing though.

--- 
I definitely see what you're saying about the facility findings, but need to think a little more on how to rephrase them to get the point across.  The major finding is definitely that there isn't a NEED for anyone to be able to be able to select or deselect a facility. The secondary finding is that if it worked in a particular way, some people would find it useful.

Maybe I don't even need to capture the second findings. My thinking in including is that these facility checkboxes keep coming up, and if they do again in the future, I don't want this information to be lost.

**Comment by andaleliz** [comment]
Hi @Samara-Strauss - I ended up [combining them into one finding](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/mhv-profile-research/mhv-user-interviews/topline-summary.md#the-participants-who-receive-care-at-multiple-facilities-did-not-have-a-need-to-select-or-deselect-facilities-in-their-my-va-treating-facilities-list) and focusing on the main point. The finding about what participants thoughts "maybe be useful if.." is included as a note under the main finding. I don't think it'll get lost, and now feels more appropriate to me as a secondary point.

I removed the additional findings section and bumped those up under interview findings, and then also added a "Next Steps" section (linked to a placeholder draft) to add some closure to the doc.

Hopefully this is all more clear. LMK what you think. Thanks!
